forge-gui/res/cardsfolder/upcoming/elspeths_smite.txt

```diff
@@ -0,0 +1,6 @@
+Name:Elspeth's Smite
+ManaCost:W
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature.attacking,Creature.blocking | TgtPrompt$ Choose target attacking or blocking creature | NumDmg$ 3 | RememberDamaged$ True | ReplaceDyingDefined$ Remembered.Creature | SubAbility$ DBCleanup | SpellDescription$ CARDNAME deals 3 damage to target attacking or blocking creature. If that creature would die this turn, exile it instead
```

Comment:
Remembering seems wrong

Reply:
Still wrong

Reply:
thanks, also fixed Annihilating Fire (where I based it on)